CrossCompileHowto.txt

```diff
@@ -63,7 +63,7 @@ echo $HOME/crosscompile/Qt5
 ##### End of Qt5 specific ######
 
 # get the Hyperion sources
-git clone --recursive https://github.com/hyperion-project/hyperion.ng.git "$HYPERION_DIR"
+git clone --recursive https://github.com/SputnikElf/hyperion.ng.git "$HYPERION_DIR"
```

Comment:
Could you please change this back?

Reply:
@b1rdhous3: not sure, if it is possible to pick commits selectively... In my PR #654 I replace the cross-compile.txt file anyway by a pretty-print .md version.